pixeltable/env.py

```diff
@@ -227,11 +242,28 @@ def _create_nos_client(self) -> None:
             func.FunctionRegistry.get().register_module(mod)
 
     def _create_openai_client(self) -> None:
-        if not 'OPENAI_API_KEY' in os.environ:
+        if 'openai' in self._config and 'default' in self._config['openai'] and 'api_key' in self._config['openai']['default']:
```

Comment:
what's the structure of the yaml file?

Reply:
Here's what I have right now:

```openai:
  default:
    api_key: ...
together:
  default:
    api_key: ...```

`default` is to allow for the possibility in the future of multiple concurrent client instances (with different API keys)

Reply:
Why have more than one api key? For higher throughput?

What are the names good for? (ie, why not make it a list of keys, if we want to support multiple keys)

Reply:
Per conversation with @mkornacker I simplified the config format to:
```
openai:
  api_key: ...
together:
  api_key: ...
```